solr/contrib/ltr/src/java/org/apache/solr/ltr/feature/impl/FieldLengthFeature.java

```diff
@@ -121,11 +121,12 @@ public FieldLengthFeatureScorer(FeatureWeight weight,
 
       @Override
       public float score() throws IOException {
+        if (norms == null) {
```

Comment:
Does it make sense to just construct and return a constant 0 value feature scorer if this is true, instead of checking for all score() calls?